chsdi/views/loader.py

```diff
@@ -1,27 +1,45 @@
 # -*- coding: utf-8 -*-
 
+import re
 import json
 from pyramid.view import view_config
 from pyramid.renderers import render_to_response
 from pyramid.request import Request
 
+regex = r"\d\.\d+\.\d+"
+
 
 @view_config(route_name='ga_api', renderer='json')
 def loadjs(request):
     mode = request.params.get('mode')
     # Determined automatically in subscriber
     lang = request.lang
+    app_version = request.registry.settings.app_version
 
     path = '/rest/services/api/MapServer/layersConfig?lang=' + lang
     subRequest = Request.blank(path)
     resp = request.invoke_subrequest(subRequest)
     data = json.loads(resp.body)
+    version_str = request.params.get('version')
+    version = ''
+    if version_str is not None:
+        m = re.match(regex, version_str)
+        if m is not None:
+            version = m.group()
```

Comment:
What happens if the version asked doesn t match any existing version -> I would expect errors after the loader is called